Marlin/pins_RIGIDBOARD.h

```diff
@@ -125,7 +125,7 @@
 #elif ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER)
 
   #undef BEEPER_PIN
-  #define BEEPER_PIN -1
+  #define BEEPER_PIN 37
```

Comment:
Actually, the solution here is to just remove this `#undef` and override altogether. The `pins_RAMPS.h` file will have set this pin to 37 already in this case.